pkiinit/Dockerfile

```diff
@@ -23,10 +23,16 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
 RUN set -eux; \
     apk add --no-cache ca-certificates dumb-init
 
+RUN mkdir -p /edgex/pkiinit
+
+WORKDIR /edgex
+
 # setup / install the executable (binary or shell)
-COPY pki-init.sh pki-init.sh
-RUN chmod +x pki-init.sh
+COPY pki-init.sh pkiinit/pki-init.sh
```

Comment:
Should be able to get rid of pki-init.sh at this time.

Reply:
sure, will remove it